File: CS101计算机编程基础系列/03_分支语句相关知识/ProgLanV2/src/proglanv2/ProgLanV2.java
package proglanv2;

import java.util.Scanner;

/**
 *
 * @author Sam_Yan
 */
public class ProgLanV2 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        System.out.print("请输入大一学的编程语言：");
        Scanner input = new Scanner(System.in);
        String progLan =  input.next();
        
        switch(progLan){
            case "C":
            case "C++":
                System.out.println("系统架构专家");
            break;
            
            case "Java":
                System.out.println("\"服务架构专家，或者多终端程序员\"");
            break;
            
            case  "Python":
                System.out.println("AI和数据分析大师");
            break;
            
            default:
                System.out.println("试试前端？");
        }
        
    }
    
}

File: CS101计算机编程基础系列/03_分支语句相关知识/PyDemo/ProgLanV2.py
# -*- coding: utf-8 -*-
"""
Created on Sun Jul 16 20:50:34 2023

@author: Sam_Yan
"""

progLan = input("请输入你大一学的编程语言：")

match progLan:
    case "C" | "C++":
        print("系统架构专家")
    case "Java":
        print("服务架构专家，或者多终端程序员")
    case "Python":
        print("AI和数据分析大师")
    case _:
        print("试试前端？")
            
    